src/dispatcher.py

```diff
@@ -44,7 +44,7 @@ def reply(self, *messages, prefix_nick=False, **kwargs):
         if self.private:
             self.source.send(*messages, **kwargs)
         elif (self.target is channels.Main and self.game_state is not None and 
-                (self.source not in get_players(self.game_state) or
+                ((self.game_state.current_phase != "join" and self.source not in get_players(self.game_state)) or
```

Comment:
I'd just get rid of this branch entirely. I can't think of any legitimately useful purpose to have it, and the nightchat one will only ever be relevant if the channel modes are misconfigured.